### PR TITLE
Implement string and console.log

### PR DESCRIPTION
**Pull request by mohanson**
E.g.

```ts
let s = "World!";
console.log("Hello %s\n", s);
```

Note: `console.log` is an alias of c languages's `stdio::printf()`

### CODE REVIEW

**Comment by yejiayu** [comment]
Please post the generated LLVM IR.

**Comment by mohanson** [comment]
```
; ModuleID = 'main'
source_filename = "main"
target datalayout = "e-m:e-i64:64-f80:128-n8:16:32:64-S128"
target triple = "x86_64-unknown-linux-gnu"

@string = private unnamed_addr constant [7 x i8] c"World!\00", align 1
@string.1 = private unnamed_addr constant [10 x i8] c"Hello %s\0A\00", align 1

declare i64 @printf(i8*, ...)

define i64 @main() {
body:
  %s = alloca i8*
  store i8* getelementptr inbounds ([7 x i8], [7 x i8]* @string, i32 0, i32 0), i8** %s
  %0 = load i8*, i8** %s
  %1 = call i64 (i8*, ...) @printf(i8* getelementptr inbounds ([10 x i8], [10 x i8]* @string.1, i32 0, i32 0), i8* %0)
  ret i64 0
}

```

File: src/codegen/index.ts [hunk 1]
```diff
@@ -206,9 +219,21 @@ export default class LLVMCodeGen {
...
+      if (item.kind === ts.SyntaxKind.Identifier) {
+        return this.genAutoDereference(this.genExpression(item));
+      } else {
+        return this.genExpression(item);
+      }
     });
-    const func = this.module.getFunction(name)!;
+    let func: llvm.Function;
+    switch (name) {
+      case 'console.log':
```
Comment (yejiayu) [inline src/codegen/index.ts]: Should `console` be considered a module?

Comment (mohanson) [reply src/codegen/index.ts]: When we go into the stage for implementing the build in functions, consider this question again. Now this is enough.

**Pull request merged**